nixos/modules/services/databases/lldap.nix

```diff
@@ -111,9 +143,9 @@ in
         ExecStart = "${lib.getExe cfg.package} run --config-file ${format.generate "lldap_config.toml" cfg.settings}";
         StateDirectory = "lldap";
         WorkingDirectory = "%S/lldap";
-        User = "lldap";
-        Group = "lldap";
-        DynamicUser = true;
+        User = cfg.user;
+        Group = cfg.group;
+        DynamicUser = cfg.dynamicUser;
```

Comment:
I feel like just adding these options doesn't buy us much. A module user will still have to understand the concept of systemd dynamic users, in order to know that they need to set it to `false` in order to interact with the service user. And at that point, they might just as easily reach into `systemd.services.lldap.serviceConfig` with `mkForce`.

Reply:
Sure, why make it easy and just expose the option with documentation when there is an workaround that the user has completely to figure out themselves without any hints or examples on search.nixos.org? /s

Reply:
Honestly, I appreciate that you're marking out your own sarcasm here, but this is still not something I need in my life as part of my volunteer work. When you decide to engage with my feedback in a straightforward way, I'll be here for it.